android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -184,6 +183,72 @@ fun purchasePackage(
     }
 }
 
+fun purchaseSubscriptionOption(
+    activity: Activity?,
```

Comment:
Hmm I know this was like this from the other purchase methods, but I wonder why make this nullable... Any context @vegaro ?